src/elements/content-sidebar/__tests__/BoxAISidebar.test.tsx

```diff
@@ -16,4 +16,10 @@ describe('elements/content-sidebar/BoxAISidebar', () => {
 
         expect(screen.getByRole('heading', { level: 3, name: 'Box AI' })).toBeInTheDocument();
     });
+
+    test('should have accessible "Expand" button', () => {
+        renderComponent();
+
+        expect(screen.getByRole('button', { name: 'Expand' })).toBeInTheDocument();
```

Comment:
Can we add `data-testid` to it and find element by this id so we won't need to update the test later on when the copy changes?
https://testing-library.com/docs/queries/bytestid/

Reply:
I mean it's a question to @jankowiakdawid

Reply:
Check out https://testing-library.com/docs/queries/about#priority

tl;dr
there are three types of queries
1. based on accessibility - ie `*byRole` `*byText` `*byDisplayValue`
2. based on semantics - `*byAltText`, `*byTitle`
3. based on testid - `*byTestId`

It's always preferable to use accessible queries as they also check if our UI is accessible.
`data-testid` is an escape hatch in rare cases where it's inconvenient to find element of interest with accessible/semantic queries.

Reply:
Thanks Dawid!